src/pages/Balance/index.tsx

```diff
@@ -159,15 +159,11 @@ const Balance: React.FC<BalanceProps> = () => {
         processTxResult(from.rpc, result, `${KWT_SCAN}/tx/${result.transactionHash}`);
         return;
       }
-      if (from.cosmosBased) {
+      // TODO: hardcode check case USDC oraichain -> USDC noble
+      if (from.cosmosBased && filterNetwork !== 'noble-1') {
```

Comment:
I still strongly believe that we don't have to hardcode noble-1 here. If theres a technical barrier, pls disclose